src/App/Helpers.fs

```diff
@@ -98,3 +98,133 @@ module Tooltip =
         |> List.fold (fun (res: string) (oldValue, newValue) ->
             res.Replace(oldValue, newValue)
         ) res
+        
+[<RequireQualifiedAccess>]
+module Translations =
+    let enTranslations = dict["msg_iframe_loaded", "Iframe loaded"; 
+        "win_header_console", "Console";
+        "msg_initializing", "Initializing";
+        "msg_repl_name", "Fable REPL";
+        "msg_desktop_experience", "For best experience we recommend running the REPL on a desktop";
+        "btn_continue", "Continue";
+        "msg_gist_description", "Created with Fable REPL";
+        "msg_compilation_failed", "Failed to compile";
+        "msg_assemblies_load_failed", "Assemblies couldn't be loaded. Some firewalls prevent download of binary files, please check.";
+        "msg_gist_token_invalid", "An error occured when creating the gist. Is the token valid?";
+        "msg_gist_token_missing", "You need to register your GitHub API token before sharing to Gist";
+        "msg_compilation_successful", "Compiled successfuly";
+        "msg_shareable_url_ready_text", "Copy it from the address bar";
+        "msg_shareable_url_ready_title", "Shareable link is ready";
+        "msg_load_gist_error", "An error occured when loading the gist";
+        "msg_update_url_failed", "An error occured when updating the URL";
+        "msg_fatal_error", "Should not happen";
+        "msg_live_sample_text", "Live sample";
+        "msg_code_text", "Code";
+        "msg_problems", "Problems";
+        "msg_problems_info", "Problems: ";
+        "msg_invalid_iframe_error", "`%A` is not a known value for an iframe message";
+        "msg_collapse_sidebar", "Collapse sidebar";
+        "msg_widget_general", "General";
```

Comment:
Personally, I would have used a Record to force us to have translation covering all the keys. This is true, that you then can't have a partial translation implemented but I feel like this is a plus.

As this will also avoid typos in the key name which would be properties instead of strings. 

This also avoid the duplication work you did below and have intellisense for free.

What do you think?

Reply:
I can add 2 languages *UK and RU, maybe Kazakh and Turkish, but that's it. I plan at least find somebody who knows Spanish and other languages. Communication can be hard in this case. over long period of time. Please consider this.
I understand why do you want it that way. I actually misinterpret part with records and go with dict. 

I ambivalent on this, and if you insist that this is better for you/maintenance, I will rework this.

Reply:
I would prefer the record option yes please. 

If we don't know how to translate something we can always referenced to the english record. This is because the REPL don't have a lot of active development done because it serves its purpose well so I want to limit the friction or things to know when contributing.

I will be able to do French on my side.

Reply:
Can you take a look at records approach ?

Reply:
@MangelMaxime I know it's Christmas, and you maybe busy, but after that, can you take a look? Also, Merry Chrismas (even if a bit late)